tests/dualtor/test_toggle_mux.py

```diff
@@ -35,7 +35,7 @@ def test_toggle_mux_from_simulator(duthosts, active_side, toggle_all_simulator_p
     logger.info('Toggle mux active side from mux simulator')
     toggle_all_simulator_ports(active_side)
 
-    check_result = wait_until(60, 5, 2, check_mux_status, duthosts, active_side)
+    check_result = wait_until(400, 5, 2, check_mux_status, duthosts, active_side)
```

Comment:
I think it's better to change heartbeat interval because if we simply run this case, heartbeat interval is 100ms, and 400s wait time will be too long